server/platform/services/awsmeter/awsmeter.go

```diff
@@ -121,26 +127,26 @@ func (awsm *AwsMeter) GetUserCategoryUsage(dimensions []string, startTime time.T
 	return reports
 }
 
-func (awsm *AwsMeter) ReportUserCategoryUsage(reports []*AWSMeterReport) error {
+func (awsm *AwsMeter) ReportUserCategoryUsage(ctx context.Context, reports []*AWSMeterReport) error {
 	for _, report := range reports {
-		err := sendReportToMeteringService(awsm.service, report)
+		err := sendReportToMeteringService(ctx, awsm.service, report)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func sendReportToMeteringService(ams *AWSMeterService, report *AWSMeterReport) error {
+func sendReportToMeteringService(ctx context.Context, ams *AWSMeterService, report *AWSMeterReport) error {
 	params := &marketplacemetering.MeterUsageInput{
 		DryRun:         aws.Bool(ams.AwsDryRun),
 		ProductCode:    aws.String(ams.AwsProductCode),
 		UsageDimension: aws.String(report.Dimension),
-		UsageQuantity:  aws.Int64(report.Value),
+		UsageQuantity:  aws.Int32(report.Value),
```

Comment:
I'm assuming the third-party library is driving the `s/int64/int32` change?

Reply:
yes, indeed, the new sdk-v2 expects int32 here